matrix_client/api.py

```diff
@@ -254,6 +254,76 @@ def send_message_event(self, room_id, event_type, content, txn_id=None,
             params["ts"] = timestamp
         return self._send("PUT", path, content, query_params=params)
 
+    def send_call_invite(self, room_id, call_id, sdp, version, lifetime,
+                         msgtype="m.call.invite", timestamp=None):
+        """Perform PUT /rooms/$room_id/send/m.room.call.invite
```

Comment:
Looks like this is actually `m.call.invite`? https://matrix.org/speculator/spec/HEAD/client_server/unstable.html#module-voip

Yep, you've got it right in actual method; just need to fixup docstring.